reports/Michigan.md

```diff
@@ -163,6 +163,20 @@ id: mi-grandrapids-2
 * https://twitter.com/sparrowsongs_/status/1266948928966139904
 
 
+### Police assault peaceful protesters, among them Breonna Taylor's family members | July 12th
+
+City of Grand Rapids Police Department officers seen pre-emptively pushing, pulling, and shoving peaceful protesters, as they circle around an officer arresting or detaining a man.
```

Comment:
Minor thing but you can probably remove the "pre-emptively"  and simplify "arresting or detaining" to just "arresting." It's kinda splitting hairs at that point and from the commentary, it looks like the protestor was arrested.